app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,44 @@
+class DownvotesController < ApplicationController
+  def create
+    @review = Review.find(params[:review_id])
```

Comment:
Wowzers! This is a big method 

This is not something that we can generally let into the codebase. Lets chat about ways we can break this up.